CityOfDetroit/bloom
# style: Arabic RTL layout changes

## Issue #1121

## Description

This PR addresses the layout issue when a user has chosen Arabic as their language - page header titles and one of the listing info containers was so not switching sides. 

## How Can This Be Tested/Reviewed?

Note: Make sure 'ar' is listed in your public .env LANGUAGES

Entry point 1: http://localhost:3000/about
Choose Arabic as your language
Notice that the page header is on the right instead of left side of the page
![Screen Shot 2022-04-22 at 1 15 39 PM](https://user-images.githubusercontent.com/60076088/164771642-bb958480-c273-4003-be57-2ac483f15663.png)

Entry point 2: http://localhost:3000/listings
Choose Arabic
Click any listing
Notice that the info container is on the left side of the listing image as opposed to the right 
![Screen Shot 2022-04-22 at 1 17 06 PM](https://user-images.githubusercontent.com/60076088/164771823-4690d210-6b3d-4bd0-82cf-7daa2abf6c07.png)


NOTE: This issue only addresses the functional layout of the page. I think it would be worthwhile to create another issue(s) that addresses some funky margin issues by swapping some of the tailwind css tags with the tailwind rtl css tags, i.e. so that an element that normally has margin-right, would then have margin-left when the layout switched to right-to-left.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
